apps/Hangover/description

```diff
@@ -0,0 +1,10 @@
+Hangover runs simple Win64 and Win32 applications on arm64 Linux. Only the application is emulated instead of a whole Wine installation.
+
+Examples on how to run:
+
+$ wine your_x86_64_application.exe # Uses FEX for 64-bit apps
+$ wine your_x86_application.exe # To use Box64 for 32-bit apps
+$ HODLL=libwow64fex.dll wine your_x86_application.exe # To use FEX for 32-bit apps
+$ HODLL=xtajit.dll   taskset -c 1 wine your_x86_application.exe # To use QEMU for 32-bit apps
```

Comment:
this seems to differ from your own description

https://github.com/AndreRH/hangover?tab=readme-ov-file#running

which is incorrect? because from your repo's description I would expect QEMU for 32bit apps to be selected with
```
HODLL=wowarmhw.dllwine your_x86_application.exe
```

Additionally, are there plans to make QEMU and Box64 options for the 64-bit emulator dll?

Reply:
thanks for having a look, I added a note
64-bit support using Box64, yes, but I'm quite time limited, so it might take a while
QEMU just isn't worth the effort